restaurants.json

```diff
@@ -0,0 +1,9 @@
+{
+  "restaurants":{
```

Comment:
it is more common to represent such data as a `list` not a `dictionary`. every JSON record should be self-contained, i.e. it is clear what means what without extra context.

 imagine I gave you data like `[12, 13, 28]` - do you have an idea about what these numbers mean?
instead, it can be `{"month": 12, "numberOfCandies": 13, "age": 28}`


so the common way to write such data is like
```json
{
  "restaurants": [
    {
      "id": 0,
      "name": "Vasilki",
      "location": "Minsk"
       ....
    }
  ]
}
```

please use a format like this to store the data in JSON and in Python

Reply:
fixed